src/data/gitmojis.json

```
@@ -431,7 +431,14 @@
 			"entity": "&#127991;",
 			"code": ":label:",
 			"description": "Adding or updating types (Flow, TypeScript)",
-			"name": "label"		
+			"name": "label"
+		},
+		{
+			"emoji": "💂",
+			"entity": "&#128130;",
+			"code": ":gard:",
+			"description": "Adding PR or Issue Template",
```

```suggestion
			"description": "Adding PR or issue template",
```
No need for capitalization here